metapool/tests/data/scrubbable-sample-sheet.csv

```diff
@@ -807,10 +807,10 @@ Lane,Sample_ID,Sample_Name,Sample_Plate,well_id_384,I7_Index_ID,index,I5_Index_I
 1,EP479894B04,EP479894B04,NYU_BMS_Melanoma_13059_P4,P24,iTru7_107_08,AGCTAACC,iTru5_124_H,AACGCCTT,NYU_BMS_Melanoma_13059,pool1,EP479894B04
 ,,,,,,,,,,,
 [Bioinformatics],,,,,,,,,,,
-Sample_Project,QiitaID,BarcodesAreRC,ForwardAdapter,ReverseAdapter,HumanFiltering,,,,,,
-NYU_BMS_Melanoma_13059,13059,False,AACC,GGTT,False,,,,,,
-Feist_11661,11661,False,AACC,GGTT,False,,,,,,
-Gerwick_6123,6123,False,AACC,GGTT,False,,,,,,
+Sample_Project,QiitaID,BarcodesAreRC,ForwardAdapter,ReverseAdapter,HumanFiltering,library_construction_protocol,experiment_design_description,,,,
+NYU_BMS_Melanoma_13059,13059,False,AACC,GGTT,False,fastprep,shotgun,,,,
+Feist_11661,11661,False,AACC,GGTT,False,fastprep,shotgun,,,,
+Gerwick_6123,6123,False,AACC,GGTT,False,fastprep,shotgun,,,,

```

Comment:
Should these names be changes package wide?

Reply:
Just for clarity here. In the SPP we changed the Sample_Project to StudyA, etc so there are no actual names in the tests, like Gerwick, etc. Just making sure that is not desirable here.